### ISSUE TITLE
[Request] Share your custom filetype highlights

### ISSUE THREAD

**Issue by olimorris** [olimorris/onedarkpro.nvim#24]
After merging `insert commit here`, the theme now has the ability to specify highlight groups by filetype. Creating this issue for us to share our custom configs to get filetypes looking as close to `One Dark Pro` as possible.

After pasting some `yaml` and `ruby` code into VS Code and comparing the colors, this is where I've ended up:

## Yaml

```lua
filetype_hlgroups = {
  yaml = {
    TSField = { fg = "${red}" } 
  }
}
```

## Ruby

```lua
filetype_hlgroups = {
  ruby = {
    TSParameter = { fg = "${fg}" },
    TSSymbol = { fg = "${cyan}" }
  }
}
```

**Comment by amaanq** [comment]
for Rust, at least what I've found "jarring" so far (sidenote: it was really hard to find "TSField", and its really unclear where this full list of TS objects are, as well as what ts rust uses...)

```lua
ft_highlights = {
    rust = {
        TSField = { fg = "${red}" },
    },
}
```

**Comment by olimorris** [comment]
@amaanq there is a section in the readme for how you find Treesitter highlight groups. You may need to play around with a few highlight groups to get it how you want

**Comment by amaanq** [comment]
> @amaanq there is a section in the readme for how you find Treesitter highlight groups. You may need to play around with a few highlight groups to get it how you want

Yeah that's fine for more generic highlight groups like TSField, TSConstructor, etc... but I want to change rust-specific ones, for example the ? is known as the try_expression when checking with the playground, I have no idea how to change that, and it might be that I have to create my own highlights.scm file? or something like that...thanks for any help with this!

**Comment by olimorris** [comment]
@amaanq pop it in a new issue and let's discuss.
